src/main/java/blue/lapis/pore/impl/PoreServer.java

```diff
@@ -767,4 +775,65 @@ public void sendPluginMessage(Plugin source, String channel, byte[] message) {
 
         return result;
     }
+    
+    
+    @SuppressWarnings("unchecked")
+    public void registerCommands() {
+        org.bukkit.plugin.Plugin[] bPlugins = this.getPluginManager().getPlugins();
+
+        for(org.bukkit.plugin.Plugin bPlugin : bPlugins) {
+        	if(bPlugin.getDescription().getCommands() != null) {
+        		getLogger().info(String.format("Loading commands for %s", bPlugin.getDescription().getName()));
```

Comment:
This could be a little spammy if a fair number of Bukkit plugins are installed; I would advise removing it.